common/src/main/kotlin/us/wedemy/eggeum/android/common/ui/BaseBottomSheetFragment.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Designed and developed by Wedemy 2023.
+ *
+ * Licensed under the MIT.
+ * Please see full license: https://github.com/Wedemy/eggeum-android/blob/main/LICENSE
+ */
+
+package us.wedemy.eggeum.android.common.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.viewbinding.ViewBinding
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+
+abstract class BaseBottomSheetFragment<VB : ViewBinding> : BottomSheetDialogFragment() {
+  private var _binding: VB? = null
+  protected val binding: VB
+    get() = _binding!!
+
+  abstract fun getViewBinding(): VB
+
+  override fun onCreateView(
+    inflater: LayoutInflater,
+    container: ViewGroup?,
+    savedInstanceState: Bundle?,
+  ): View {
+    _binding = getViewBinding()
+    return binding.root
+  }
```

Comment:
```suggestion
  ) = getViewBinding().also { _binding = it }.root
```

예전부터 잡고 싶었어서 이번 기회에 말해드립니당 😄

Reply:
좋네요

Reply:
`also`가 엄청난 친구입니다 호호
전 scope function 중에서 `also`를 제일 좋아해요 ㅎ.ㅎ